src/libs/OptionsListUtils.ts

```diff
@@ -992,7 +992,7 @@ function sortCategories(categories: Record<string, Category>): Category[] {
     const sortedCategories = Object.values(categories).sort((a, b) => a.name.localeCompare(b.name));
 
     // An object that respects nesting of categories. Also, can contain only uniq categories.
-    const hierarchy = {};
+    const hierarchy: Hierarchy = {};
```

Comment:
So we can do `var: type = something` ? Seems like you delete it everywhere else.

Reply:
It's a different case actually, because without `Hierarchy` type here hierarchy variable accepts only an empty object `{}` type which is incorrect.